kitsune/search/v2/search.py

```diff
@@ -195,6 +202,46 @@ def make_result(self, hit):
         }
 
 
+class ForumSearch(SumoSearch):
+    """Search over User Profiles."""
+
+    def get_index(self):
+        return ForumDocument.Index.read_alias
+
+    def get_fields(self):
+        return ["thread_title", "content"]
+
+    def get_highlight_fields(self):
+        return ["thread_title", "content"]
+
+    def get_filter(self, **kwargs):
+        # Add default filters:
+        filters = [
+            # limit scope to the Wiki index
+            DSLQ("term", _index=self.get_index())
+        ]
+
+        if thread_forum_id := kwargs.get("thread_forum_id"):
+            filters.append(DSLQ("term", thread_forum_id=thread_forum_id))
+        return DSLQ("bool", filter=filters, must=self.build_query(**kwargs))
+
+    def get_highlight_options(self):
+        return {}
+
+    def make_result(self, hit):
+        return {
+            "type": "thread",
+            "title": hit.thread_title,
+            "search_summary": bleach.clean(wiki_to_html(hit.content[:1000]), strip=True),
```

Comment:
Ah, I was wrong, our bleach for other results is a little different, we can just use the `strip_html` function. Additionally we ought to slice after, I saw oddities with a few results because we cut partway through wiki syntax:

```python
strip_html(wiki_to_html(hit.content))[:1000]
```